src/window/empire.c

```diff
@@ -413,6 +437,7 @@ static void draw_foreground(void)
     }
     draw_city_name(city);
     draw_panel_buttons(city);
+    draw_object_info();
```

Comment:
This was in draw_background, but with the new button handling it needed to be handled on the foreground